Snakefile

```diff
@@ -1,6 +1,7 @@
 rule target:
     input: 
-        "01-getting-started/getting-started.html" 
+        "01-getting-started/getting-started.html",
+	"03-rnaseq/clustering/clustering_heatmap_example.html" 
```

Comment:
Snakemake, being based on python, cares a lot about spacing.
```suggestion
	  "03-rnaseq/clustering/clustering_heatmap_example.html" 
```

Reply:
Ahh of course!